v2/sample/wss/main.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2018 SEQSENSE, Inc.
```

Comment:
would be nice to rename `sample` to `examples`, and put original example to an another directory (`mqtt`?).